CONTRIBUTING.md

```diff
@@ -248,7 +248,7 @@ You'll need **[Python 3.8](https://github.com/huggingface/transformers/blob/main
    messages](https://chris.beams.io/posts/git-commit/) to clearly communicate the changes you made!
 
    To keep your copy of the code up to date with the original
-   repository, rebase your branch on `upstream/branch` *before* you open a pull request or if requested by a maintainer:
+   repository, rebase your branch on `upstream/main` *before* you open a pull request or if requested by a maintainer:
```

Comment:
I'd keep this as-is - technically we can rebase on and keep up-to-date with any branch and the snippet below shows rebasing on main 

```suggestion
   repository, rebase your branch on `upstream/branch` *before* you open a pull request or if requested by a maintainer:
```